app/views/blog/feed.html.erb

```diff
@@ -0,0 +1,22 @@
+<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/">
+  <channel>
+    <title>Git - Blog</title>
+    <link><%= request.protocol %><%= request.host_with_port %></link>
+    <atom:link href='<%= request.url %>' rel='self' type='application/rss+xml' />
+    <description>Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.</description>
+
+    <% list_posts.each do |entry| %>
+      <% post = preview_post(entry[:file]) %>
+      <item>
+	<title><%= entry[:title] %></title>
+	<guid><%= format_path(entry) %></guid>
+	<link><%= format_path(entry) %></link>
+	<% if post.last['author'] %>
+	<author><%= post.last['author'] %></author>
+	<% end %>
+	<pubDate><%= entry[:date_published].rfc822 %></pubDate>
+	<description><%= post.first %></description>
+      </item>
+    <% end %>
+  </channel>
+</rss>
```

Comment:
can't find this one 🤔

Reply:
I think it's at `/blog.rss` (you have to check `config/routes.rb`). But really, it can probably just be deleted entirely. There haven't been posts in years.

Reply:
it seems to be. I was there, but the contents don't seem ok (in old and new version) https://git-scm.com/blog.rss

Reply:
Speaking of this file, I'm wondering if we could remove the blog feed since nothing has been posted in nearly 7 years.

Reply:
:thumbsup: I'll remove the template and link to it here.